### ISSUE TITLE
using cifar_10 for evaluation

### ISSUE THREAD

**Issue by ouisyasser** [tensorflow/models#8592]
Hello, I trained the cifar10_train.py model and I saved the last chechkpoints, But now Im thinking about reload the traind model and use it in order to evaluate it on my own dataset. But the problem is that I dont know how to reload the model using the last checkpoints files!

**Comment by djdongjin** [comment]
Can you provide more details? Like which version of tf you are using or the link of the file you are mentioning. Also, this [tutorial](https://www.tensorflow.org/tutorials/keras/save_and_load) has a complete go-through on how to load a model from checkpoints, SavedModel format, etc.

**Comment by ravikyram** [comment]
@ouisyasser 

It is difficult to understand your issues only based on the information you provided.Please, let us know which pretrained model you are using and share related code .Please, let us know TF version you are using.Thanks!

**Comment by ouisyasser** [comment]
I'm using wrn model. I trained it for 200 epocks and get 0.98 accuracy.I saved the last checkpoints.
The colab is showing the code I execute it the model for training 
https://colab.research.google.com/drive/1TLitkT-LghpLsZ4EJX9PRS1Q7bXOq_8I?usp=sharing

**Comment by djdongjin** [comment]
@ouisyasser Is this still an issue?

**Comment by djdongjin** [comment]
Closed since there is no response. Feel free to open again.
